osde2e/managed_cluster_validating_webhooks_test.go

```diff
@@ -603,3 +632,8 @@ var _ = Describe("Managed Cluster Validating Webhooks", Ordered, func() {
 		})
 	})
 })
+
+// Helper function to create a pointer to a bool
+func boolPtr(b bool) *bool {
+	return &b
+}
```

Comment:
All usage of this can be replaced with the builtin K8s `pointer` function `pointer.BoolPtr(true)`